pkg/azdiskschedulerextender/main.go

```diff
@@ -42,6 +43,7 @@ func init() {
 var (
 	metricsAddress              = flag.String("metrics-address", "0.0.0.0:29606", "export the metrics")
 	azDiskSchedulerExtenderPort = flag.String("port", "8889", "port used by az scheduler extender")
+	driverName                  = flag.String("drivername", consts.DefaultDriverName, "name of the driver")
```

Comment:
We should make sure this is added to our scheduler config in aks

Reply:
It is for now the same as the default driver name so we can modify the chart when necessary.

Reply:
This will break with side-by-side setup, so we do need to modify it

Reply:
I thought you were referring to the scheduler extender chart in AKS rp.

But yes, we will need to update the helm chart in the main_v2 repo.

Reply:
Yep, I was initially and then I remembered about the side-by-side setup :D I think we should pass this value explicitly in both places because it will break the scheduler extender's logic if the driver gets deployed under a different name

Reply:
Sounds good.